frontend/src/components/Player/Leaders.vue

```diff
@@ -15,7 +15,7 @@
             </span>
             <div style="margin-top: 10px;">
               <div v-for="(leader, index) in filteredAllTimePointLeaders" v-bind:key="leader.id" class="row-item">
-                <router-link :to="'/player/' + leader.id + '/info'">{{ index+1 }}. {{ leader.name }}</router-link>
```

Comment:
Please don't revert my fantastic feature :pray:

Reply:
that's just the diff, I didn't touch this file at all :dagger: